### PR TITLE
Add Laravel 5.8 support

### PR DESCRIPTION
**Pull request by dabich**

### CODE REVIEW

**Comment by davejamesmiller** [comment]
Tests seem to be failing so I can't merge it - but not just in the latest version, so I'm not sure why - but I don't have time to investigate right now.

**Comment by ammonkc** [comment]
Looks like there's a missing comma in composer.json on line 18

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ A simple [Laravel](https://laravel.com/)-style way to create breadcrumbs.
 
 | Laravel Breadcrumbs                                                    | Laravel   | PHP  |
 |------------------------------------------------------------------------|-----------|------|
-| **5.1.1+**                                                             | 5.6 – 5.7 | 7.1+ |
+| **5.1.1+**                                                             | 5.6 – 5.8 | 7.1+ |
```
Comment (davejamesmiller) [inline README.md]: BTW this line is incorrect - 5.8 was not supported in 5.1.1.

```suggestion
| **5.2.1+**                                                             | 5.6 – 5.8 | 7.1+ |
| 5.1.1-5.2.0+                                                           | 5.6 – 5.7 | 7.1+ |
```

**Comment by ammonkc** [comment]
I'm not sure if this is what's causing it to fail, but the deprecated `array_first()` helper method is being use here:
https://github.com/davejamesmiller/laravel-breadcrumbs/blob/8e2ddef22bc0451f7b1df714c0de3c8ec20ec843/src/BreadcrumbsManager.php#L282

It should probably be updated to `Arr::first()`

According to the Laravel docs this helper method is deprecated:
> All array_* and str_* global helpers have been deprecated. You should use the Illuminate\Support\Arr and Illuminate\Support\Str methods directly.

**Comment by davejamesmiller** [comment]
> It should probably be updated to `Arr::first()`

Good idea.

**Comment by lloricode** [comment]
phpunit `setUp()` and `tearDown()` need to add return type `:void`

**Comment by KubqoA** [comment]
I think it isn't enough to just rollback the version of phpunit to 7 while leaving the void return type intact in the setup methods. I feel like you need to modify that as well. I would do it myself but I don't have the write privileges.

**Comment by davejamesmiller** [comment]
Good catch on the `.travis.yml` file. I'm just running some local tests then will merge & release it. Thanks!

**Comment by davejamesmiller** [comment]
All done.

**Pull request merged**